Cargo.toml

```diff
@@ -25,32 +25,32 @@ default-logger = ["slog-stdlog", "slog-envlogger", "slog-term"]
 
 # Make sure to synchronize updates with Harness.
 [dependencies]
+fxhash = "0.2"
+fail = { version = "0.3", optional = true }
+getset = "0.0.9"
 protobuf = "2"
-slog = "2.2"
-slog-stdlog = { version = "4", optional = true }
-slog-envlogger = { version = "2.1.0", optional = true }
-slog-term = { version = "2.4.0", optional = true }
-quick-error = "1.2.2"
+quick-error = "1"
 raft-proto = { path = "proto", version = "0.6.0-alpha", default-features = false }
 rand = "0.7"
-fxhash = "0.2.1"
-fail = { version = "0.3", optional = true }
-getset = "0.0.7"
+slog = "2"
```

Comment:
I suggest to keep all these versions as their original values. For example, it's different between "2" and "2.2". Libraries work with 2.2.0 don't have to work with 2.0.0. Changing it to 2 gives a false assumption that we have tested and guarantee compatible against 2.1 and 2.0.